src/indra_cogex/client/enrichment/continuous.py

```diff
@@ -390,13 +419,13 @@ def indra_upstream_gsea(
 
 @autoclient()
 def indra_downstream_gsea(
-    scores: Dict[str, float],
-    directory: Union[None, Path, str] = None,
-    *,
-    client: Neo4jClient,
-    minimum_evidence_count: Optional[int] = None,
-    minimum_belief: Optional[float] = None,
-    **kwargs,
+        scores: Dict[str, float],
```

Comment:
We can revert all the spurious style changes

Reply:
8 space indent reverted in 2ccebf5